content/events/launchpad-summit-ethcc.md

```diff
@@ -0,0 +1,37 @@
+---
+ eventName: "Launchpad Summit Paris" #the name of your track or event and its mandatory
+
+ eventDescription: "Launchpad is a blend of two key components:  a dynamic four-week virtual learning cohort, where residents actively participate in remote learning seminars, and an unforgettable one-week in-person "colo" Summit.  These summits comprise interactive presentations, technology deep dives, and unparalleled networking opportunities." #short description of your track or event limiting to 100-150 characters
```

Comment:
@laeccles Please update the content in the description's last line - if the term colo is required to be enclosed within double quotations, use the escape character \ before the double quotations just like below -
BACKSLASH"coloBACKSLASH"

Reply:
@WinsCodes I have updated this.